### ISSUE TITLE
"use strict" error

### ISSUE THREAD

**Issue by ComFreek** [UltCombo/traceuroso#3]
`bootstrap.js`:

```
'use strict';

module.exports = require('traceuroso')(__dirname, 'index');
```

`index.js`:

```
'use strict';

module.exports = {};
```

Error output:

```
P:\node_modules\traceuroso\index.js:21
                        throw new Error(compileErrPrefix + results.errors[0]);
                                                                         ^
TypeError: Cannot read property '0' of undefined
    at Object.Module._extensions..js (P:\node_modules\traceuroso\index
.js:21:53)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at module.exports (P:\node_modules\traceuroso\index.js:39:9)
    at Object.<anonymous> (P:\bootstrap.js:1:101)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)

P:\>
```

If I change [this line in traceuroso/index.js](https://github.com/UltCombo/traceuroso/blob/3446d25c5e0ee37d608bc9f259bc9504f456bcca/index.js#L21) to:

```
throw new Error(compileErrPrefix + results);
```

the error output is as follows:

```
P:\node_modules\traceuroso\index.js:21
      throw new Error(compileErrPrefix + results);
            ^
Error: traceuroso: traceur.compile error: "use strict";
module.exports = {};
//# sourceURL=<compile-source>
    at Object.Module._extensions..js (P:\node_modules\traceuroso\index
.js:21:13)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at module.exports (P:\node_modules\traceuroso\index.js:38:9)
    at Object.<anonymous> (P:\bootstrap.js:1:101)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)

P:\>
```

The error also occurs when completely removing the `use strict` statements from bootstrap.js and index.js

Is this issue related to Traceuroso or Traceur?

System information:
- OS: Windows 8 Pro 64-bit
- Node.js: 0.12.0 (also tested with iojs 1.2.0)
- Traceuroso: 0.0.4
- Traceur: 0.0.85

**Comment by UltCombo** [comment]
Dupe of #2 I guess. The problem is, Traceur keeps making breaking changes to the API without notice.

Traceur does not actually follow semantic versioning, it has only `0.x` version releases which are not covered by semver. This means that there are two ways to register Traceur as a dependency of this package:

1. Depend on a fixed version of Traceur (the most "correct" approach). The problem is that Traceur pushes new releases very fast, and the users would be locked in an outdated version of Traceur until a maintainer of traceuroso updated the dependency version manually.

2. Depend on the latest version of Traceur and fix things when they break. This should work more seamlessly for the users.

I went for approach 2, and I admit I really did not foresee so many API breaking changes in Traceur.

I'll fix this issue and then deprecate traceuroso in favor of [slush-es20xx](https://github.com/es6rocks/slush-es20xx), which provides a full development and deployment ECMAScript.next workflow.

**Comment by UltCombo** [comment]
fixed by aee0f389465e2c1e25a164991efd6e4d27a890f0, released in traceuroso v0.0.5.
Also, traceuroso is now deprecated in favor of [slush-es20xx](https://github.com/es6rocks/slush-es20xx).

**Comment by ComFreek** [comment]
:+1: Thanks!
